SmartDeviceLink/SDLTouchManagerListener.h

```diff
@@ -0,0 +1,25 @@
+//
+//  SDLTouchManagerListener.h
```

Comment:
If that's the case, we should be modifying all other Protocols that use `-Listener`, I was simply trying to keep it consistent 👎